mcs/class/System.Data/Test/ProviderTests/System.Data.SqlClient/SqlConnectionTest.cs

```diff
@@ -287,6 +287,9 @@ public void Close ()
 		[Test]
 		public void ChangeDatabase ()
 		{
+			if (ConnectionManager.Instance.Sql.IsAzure)
+				Assert.Ignore("SQL Azure doesn't support 'ChangeDatabase'");
```

Comment:
we could maybe verify that an exception is thrown from `conn.ChangeDatabase()` in the Azure case.